ibis/util.py

```diff
@@ -355,3 +356,39 @@ def flatten_iterable(iterable):
             yield from flatten_iterable(item)
         else:
             yield item
+
+
+def warn_deprecated(name, ver='', instead='', ignoreframes=1):
```

Comment:
I think `ver` (`version` ideally) and `instead` should be required keyword arguments similar to the decorator.